.distignore

```diff
@@ -4,6 +4,7 @@
 /.wordpress-org
 /node_modules
 /tests
+/vendor
```

Comment:
I think this needs to be removed since its needed for the BaseGateway class, and then in the current branch as well for the payment gateways as well.

Reply:
Good catch!